.pre-commit-hooks.yaml

```diff
@@ -4,33 +4,39 @@
   entry: fix-smartquotes
   language: python
   types: [text]
+- id: fix-unicode-dashes
+  name: Fix Unicode dash characters
+  description: "Replace Unicode dash characters (en-dash, em-dash, etc.) with ASCII hyphens"
+  entry: fix-unicode-dashes
+  language: python
+  types: [text]
 - id: fix-ligatures
   name: Fix ligature characters with NFKD normalization
-  description: 'Replace ligature characters with normalized individual characters'
+  description: "Replace ligature characters with normalized individual characters"
   entry: fix-ligatures
   language: python
   types: [text]
 - id: fix-spaces
   name: 'Normalize irregular space characters to "space"'
-  description: 'Replace non-breaking spaces and other characters with the standard space character'
+  description: "Replace non-breaking spaces and other characters with the standard space character"
   entry: fix-spaces
   language: python
   types: [text]
 - id: forbid-bidi-controls
   name: Forbid the use of unicode BiDi control characters
-  description: 'Check for lines of text which contain bidirectional text control characters'
+  description: "Check for lines of text which contain bidirectional text control characters"
   entry: forbid-bidi-controls
   language: python
   types: [text]
 - id: macro-expand
   name: Expand text macros
-  description: 'Perform simple macro replacements in text files'
+  description: "Perform simple macro replacements in text files"
   entry: macro-expand
   language: python
   types: [text]
 - id: alphabetize-codeowners
   name: Alphabetize Codeowners
-  description: 'Alphabetize GitHub CODEOWNERS files to list owners in the same order'
+  description: "Alphabetize GitHub CODEOWNERS files to list owners in the same order"
```

Comment:
Could you revert these quotation mark changes? They are non-semantic and will dirty git-blame for each of these lines.